.gitignore

```diff
@@ -15,6 +15,7 @@ eshell/lastdir
 my-org/
 org-files/
 semanticdb/
+srecode-map.el
```

Comment:
Hey, sorry just saw this, I had a look and I think it's something that got generated by semantic-mode. Or actually I think it's CEDET mode [from this stackoverflow question](https://stackoverflow.com/questions/16267830/how-to-move-cedet-auto-generated-ede-projects-el-and-srecode-map-el)